frontend/src/components/common/RecentlyViewed/RecentlyViewed.jsx

```diff
@@ -6,7 +6,7 @@ import { GetProducts } from '../../../services/products/productsAccess.service';
 
 const RecentlyViewed = React.memo(() => {
   const recentlyViewed = useMemo(
-    () => Array.from(JSON.parse(localStorage.getItem('recentlyViewed') || '')),
+    () => Array.from(JSON.parse(localStorage.getItem('recentlyViewed')) || {}),
```

Comment:
Could you please explain why do you use here `{}`?
'recentlyViewed' is an array in local storage

Reply:
'recentlyViewed' returned as a string. In case we don't have such a string in local storage, then JSON.parse will return an error. That's why I use {}, to prevent errors in Array.from method